VideoWeb/VideoWeb/ClientApp/src/app/shared/helpers/sorting-helper.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { SortingHelper } from './sorting-helper';
+
+describe('SortingHelper', () => {
+    it('should create an instance', () => {
+        expect(new SortingHelper()).toBeTruthy();
+    });
+
+    describe('SortingHelper', () => {
```

Comment:
suggest changing this to `describe('orderByRoleThenName', () => {`

to match the function name that you are testing

Reply:
done